sql/dj_setup_database.in

```diff
@@ -386,6 +397,41 @@ dump)
 	exit 0
 	;;
 
+load)
+	choice=""
+	DUMPNAME="$2"
+	if [ -z "$DUMPNAME" ]; then
+		set -- $(ls $DATABASEDUMPDIR)
+		ind=0
+		while true; do
+			read -p "Which database should be loaded? " db
+			ind=0
+			for i in $@; do
+				if [ "$ind" = "$db" ]; then
+					choice="$i"
+					break
+				fi
+				: $((ind+=1))
+			done
+			if [ -n "$choice" ]; then
+				break
+			fi
+		done
+	else
+		choice="${DUMPNAME}.sql.gz"
+	fi
+	FILE="$DATABASEDUMPDIR/${choice}"
+
+	if [ ! -f "${FILE}" ]; then
+		echo "Error: file ${FILE} not found."
+		exit 1
+	fi
+
+	uninstall_helper
+	create_db_users_helper
+	pv "${FILE}" | gunzip | mysql domjudge
```

Comment:
is `pv` already somewhere in the docs as requirement?

Reply:
It wasn't, but it is now.